src/maggma/stores/open_data.py

```diff
@@ -754,3 +755,35 @@ def __eq__(self, other: object) -> bool:
             "object_grouping",
         ]
         return all(getattr(self, f) == getattr(other, f) for f in fields)
+
+
+class TasksOpenDataStore(OpenDataStore):
+    """
+    Task data is stored on S3 compatible storage using the format used by Materials Project on OpenData.
+    The index is loaded from S3 compatible storage into memory.
+
+    Note that updates will only affect the in-memory representation of the index - they will not be persisted.
+    To persist index writes utilize the `store_manifest` function.
+
+    This Store should not be used for applications that are distributed and rely on reading updated
+    values from the index as data inconsistencied will arise.
+    """
+
+    def __init__(
+        self,
+        **kwargs,
+    ):
+        """Initializes a TaskOpenDataStore."""
+        super().__init__(**kwargs)
+
+    def _index_for_doc_from_s3(self, key: str) -> pd.DataFrame:
+        print(f"indexing doc with id: {key}")
```

Comment:
I have been getting screwed lately by raw print()'s when using Dask, is this print necessary? And/or could a logger be set up here?

Reply:
good catch on this. I forgot to remove it from when I was debugging.